### ISSUE TITLE
Determine graceful method for pulling up keyboard help dialog

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#122]
From #116, @terracoda mentioned that 

> The Keyboard commands and help dialog only works when balloon has focus. Ideally help should be available from anywhere. I think this was a temporary implementation of help.

The current method of bringing up the keyboard help dialog is to press the '?' key.  For most browsers and screen readers, '?' is intercepted by the AT unless in 'forms' mode.  The keyboard help dialog will only be accessible when the user has focus on a form element.

```application``` role will not be applied to the entire simulation element, so it will not be possible to pull up the keyboard help from anywhere in the sim.

@terracoda, should we restrict the cue for the keyboard help dialog so that it only applies to the balloon?  Or should we create new button on the screen that pulls up the dialog instead of a keyboard command?

**Comment by jessegreenberg** [comment]
I would recommend a separate button that pulls up the keyboard help menu.  This way, the keyboard help menu is available for those that do not use screen readers.  Perhaps it could have a unique icon that would be used for all simulations.  Alternatively this could exist in the PhET menu, but that is more difficult to access.

**Comment by jessegreenberg** [comment]
At 5/17 meeting, we discussed that a button in the footer or somewhere else could handle this,  but a hotkey would still be a valuable shortcut.

**Comment by jessegreenberg** [comment]
We can discuss adding a button later, but closing this issue for issue hygiene.  At the moment, the keyboard help dialog can be brought up with 'h' from the balloon with its application role.
